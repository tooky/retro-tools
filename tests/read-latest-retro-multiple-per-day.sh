EDITOR=fake-editor retro_date=1997-06-05 retro add
echo "- do something" >> doc/retro/1997/06/05.md
EDITOR=fake-editor retro_date=1997-06-05 retro add
echo "- do something different" >> doc/retro/1997/06/05.md
retro latest
