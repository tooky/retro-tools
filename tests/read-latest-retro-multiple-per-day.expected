EDITOR=fake-editor retro_date=1997-06-05 retro add
EDITOR doc/retro/1997/06/05.md
doc/retro/1997/06/05.md
echo "- do something" >> doc/retro/1997/06/05.md
EDITOR=fake-editor retro_date=1997-06-05 retro add
EDITOR doc/retro/1997/06/05.md
doc/retro/1997/06/05.md
echo "- do something different" >> doc/retro/1997/06/05.md
retro latest
# 1997-06-05
## Who

## What happened

## Insights

## Actions
- do something different
